src/main/java/Task.java

```diff
@@ -0,0 +1,26 @@
+public class Task {
+    protected String taskName;
+    protected boolean isDone;
+
+    public Task(String taskName) {
```

Comment:
It would be good to not have parameters and variables the same name to avoid confusion